### PR TITLE
couple bug fixes

### PR DESCRIPTION
**Pull request by qzhan16**
fix couple bugs in scripts @mestery

### CODE REVIEW

File: build/networking/scripts/build_p4sde.sh [hunk 1]
```diff
@@ -59,11 +59,11 @@ if [ -d "p4-driver" ]; then rm -Rf p4-driver; fi
 echo "Compiling p4-driver"
 #TODO: Below link needs to be updated when code is open-sourced
 git clone https://github.com/p4lang/p4-dpdk-target.git --recursive p4-driver
-cd p4-driver
+cd "$SDE/p4-driver"
```
Comment (mestery) [inline build/networking/scripts/build_p4sde.sh:62]: Please use this syntax instead:

```
pushd "$SDE/p4-driver"
git checkout 780b3dfa205815e87f4580383cc37bfa30187f7c
popd
```

File: build/networking/scripts/build_p4sde.sh [hunk 2]
```diff
@@ -59,11 +59,11 @@ if [ -d "p4-driver" ]; then rm -Rf p4-driver; fi
...
 git checkout 780b3dfa205815e87f4580383cc37bfa30187f7c
 
 pip3 install distro
-cd p4-driver/tools/setup
+cd "$SDE/p4-driver/tools/setup"
```
Comment (mestery) [inline build/networking/scripts/build_p4sde.sh:66]: Instead of `cd`, use `pushd` and `popd`.

**Review by mestery — CHANGES_REQUESTED** [review]

File: build/networking/scripts/build_p4sde.sh (hunk 1 above)
Comment (qzhan16) [reply build/networking/scripts/build_p4sde.sh]: updated in new version, thanks

**Review by mestery — APPROVED** [review]

**Comment by a37a32** [comment]
Hi @mestery , any timeline when this PR will be merged? Thanks.

**Comment by mestery** [comment]
Hi @qzhan16, can you rebase this change and then force push here so it re-triggers the CI jobs? Thanks!

**Comment by ipdksw** [comment]
https://cbjenkins-fm.devtools.intel.com/teams-nex-epgsw-ipdkci/job/nex-epgsw-ipdkci/job/GitHub_MultiBranch_Pipelines/job/ipdk/job/PR-176/ Result:ABORTED JenkinsConsole:https://s3.ipdk.io/nex-epgsw-ipdkci/GitHub_MultiBranch_Pipelines/ipdk/PR-176-6/index.html GitCommit:https://github.com/ipdk-io/ipdk/commits/9a7896fcb9fedb76227c31888224df1bf51db8ba

**Review by mestery — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**